autovalet_husky/src/autovalet_husky/find_parking_spot.py

```diff
@@ -39,6 +39,9 @@ def __init__(self, goal_topic, tag_topic, goal_frame_id, husky_frame_id, aruco_f
         self.aruco_frame_name = aruco_frame_name
         self.goal_frame_id    = goal_frame_id
         self.husky_frame_id   = husky_frame_id
+        self.first_goal_close = False
```

Comment:
remove this line. I renamed it in my earlier MR and I think it slipped through your branch when you're rebasing.

Reply:
ok done